content/python/concepts/built-in-functions/terms/print/print.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'print()'
+Description: 'Takes in data such as a string or an int converted to string using str() and prints it to the standard output device.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The print() function takes in a string or any other element such as int converted to a string using str() and displays those values to the standard output device.
+
+## Syntax
+
+```py
+print("Hello, World!")
+```
+
+The output would be:
+
+```py
+Hello, World!
+```
+
+If one were to print an integer of a floating-point number they can do so by using the str() function:
+
+```py
+print(str(3.14))
```

Comment:
This works fine, though passing just 3.14 into the function displays the implicit call of `str()`. I'd recommend to not type cast the integer to a string but we can still explain what's going on like how you've done on line 31 for the current line 33-35 codeblock.